File: app/src/main/java/in/completecourse/LoginActivity.java
package in.completecourse;

import android.os.Bundle;
import android.view.View;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentTransaction;

import in.completecourse.fragment.authFragment.EasyLoginFragment;

public class LoginActivity extends AppCompatActivity implements View.OnClickListener {


    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);
        loadFragment(new EasyLoginFragment());
    }

    @Override
    public void onClick(View v) {

    }

    /**
     * loading fragment into FrameLayout
     * @param fragment is the fragment which we want to load in our FrameLayout
     */
    private void loadFragment(Fragment fragment) {
        // load fragment
        FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
        transaction.replace(R.id.frame_container, fragment);
        transaction.commit();
    }


}

File: app/src/main/java/in/completecourse/LoginActivity.kt
package `in`.completecourse

import `in`.completecourse.fragment.authFragment.EasyLoginFragment
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment

class LoginActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)
        loadFragment(EasyLoginFragment())
    }

    /**
     * loading fragment into FrameLayout
     * @param fragment is the fragment which we want to load in our FrameLayout
     */
    private fun loadFragment(fragment: Fragment) {
        // load fragment
        val transaction = supportFragmentManager.beginTransaction()
        transaction.replace(R.id.frame_container, fragment)
        transaction.commit()
    }

}